OrbitLinuxTracing/include/OrbitLinuxTracing/Tracer.h

```diff
@@ -73,11 +72,10 @@ class Tracer {
   std::vector<Function> instrumented_functions_;
 
   TracerListener* listener_ = nullptr;
-
-  bool trace_context_switches_ = true;
-  bool trace_callstacks_ = true;
-  bool trace_instrumented_functions_ = true;
-  bool trace_gpu_driver_ = true;
+  TracingOptions tracing_options_{.trace_context_switches = true,
```

Comment:
I like this syntax a lot! But *I think* it's officially C++20. Does gcc also support it?

Reply:
I only tried it with gcc so far. Will try it with clang now.

Reply:
Seem to work with clang9 and gcc9